docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,81 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low and when the usage exceeds 90% notifies the customer.
+
+If any of the nodes in the service exceeds the critical threshold of disk usage (more than 97%), the access control list (ACL) used to authorize API requests by Apache Kafka clients is updated on all nodes to prevent operations that would increase disk usage. Such operations include:
+
+-  The ``Write`` and ``IdempotentWrite`` operations that clients use to produce new messages
+
+-  The ``CreateTopics`` operation that creates one or more topics, each carrying some overhead on disk
+
+When the ACL blocks write operations, clients see an error similar to the following example from the ``python-kafka`` client::
+
+   TopicAuthorizationFailedError: [Error 29] TopicAuthorizationFailedError: mytopic
+
+
+To ensure that enough disk space is available you can take one of the following actions:
```

Comment:
```suggestion
To ensure that enough disk space is available, you can take one of the following actions:
```

Reply:
I am trying to thinking as follow: 
1) I see the error above. Something is wrong with the space.
2) I will go over the steps to fix it. 

do we know if those errors are related to `not enough disk space available` specific?

Reply:
About the structure, I like that there is a section explaining each one of them. 

How about having the options listed with a link for each section that then while I am reading the options I can easily navigate by clicking and jumping to the section in particular? Something like [this](https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page)

```
If <describe issue>, take the following actions:
1) Upgrade to a larger service plan
2) Add additional storage space
3) Delete one or more topics
4) Decrease retention time/size
```

I am not sure if you want to use the numbers. But maybe yes, if the steps should be tried one after the other.
For the internal ones, like in the inner sections, one option is to use a) b) c) for e.g.

Reply:
We  actually have something similar and automatic already - On the right side there is content list. Is it similar to what you had in mind?
<img width="1018" alt="Screenshot 2021-12-15 at 11 46 26" src="https://user-images.githubusercontent.com/4600541/146172750-f936aaf1-94bc-418d-a3a1-c4dfaa60cdde.png">

Reply:
Not really, I thought about having it listed on the following actions: then list with a link to each section. The content helps but I was so focused on the reading that I did not see it

Reply:
In this case, It's more of a global change to how we put content. Because if we do it here, we should follow same pattern everywhere, right?

Reply:
not really, but you do not need to add in this PR. It is more a detail how the reading flow can be improved. The content tab is created automatic from the sub-headings and what I proposed is to have the steps listed in bullet points that could be linked to the text below. However, it can be improved later on as this should be merged soon 👍

Reply:
But wouldn't it mostly repeat the **contents** part in this case? :)